manifests/12_prometheusrules.yaml

```diff
@@ -25,3 +25,22 @@ spec:
             Cluster storage operator monitors all storage classes configured in the cluster
             and checks there is not more than one default StorageClass configured.
           message: "StorageClass count check is failing (there should not be more than one default StorageClass)"
+
+    - name: storage-operations.rules
+      rules:
+      - alert: PodStartupStorageOperationsFailing
+        # There was at least one failing operation in past 5 minutes *and* there was no successful one.
+        # Focus on attach and mount operations - they have the same diagnostic steps and are the most common.
+        expr: |
```

Comment:
We should perhaps add a comment that - this does not cover the cases where attach operation does not even start. For example - a node is shutdown and pod is deleted, but detach won't start until a certain check in ADC expires (I think something like 5 minutes after which volume is force detached).

Which reminds me - do we need an alert for detach failures? may be we can do that in a follow up. But if detach is failing then attach operation may not even start and hence no attach failure metric might be emitted.